doc/articles/get-started-vscode-omnisharp.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: Uno.GetStarted.vscode.OmniSharp
+---
+
+## How to switch to OmniSharp Mode
```

Comment:
Same here. It should be H1 and then all next headers reduced by one level

Reply:
same reason as earlier

Reply:
that was fixed in the other document